diff/diff.go

```diff
@@ -33,7 +33,11 @@ func diffBase(mode Mode) (string, error) {
 	gitMergeBase := exec.Command("git", "merge-base", mode.BaseBranch, "HEAD")
 	mergeBase, err := gitMergeBase.Output()
 	if err != nil {
-		return "", fmt.Errorf("cannot find merge base with branch '%s': %s", mode.BaseBranch, err.(*exec.ExitError).Stderr)
+		if ee, ok := err.(*exec.ExitError); ok {
+			return "", fmt.Errorf("cannot find merge base with branch '%s': %s", mode.BaseBranch, ee.Stderr)
+		} else {
```

Comment:
do you need the `else`?

Reply:
I think so - It's reporting a more basic error if it's not an ExitError